File: Code/Haskell/problem005/euler005.hs

euler = testdivs 20 testList

testdivs n xs
 | isEvenDivis n xs = n
 | otherwise = testdivs (n+1) xs

testList = [7,11,12,13,14,15,16,17,18,19,20]

isEvenDivis _ [] = True
isEvenDivis n (x:xs) 
 | n `rem` x == 0 = isEvenDivis n xs
 | otherwise = False

main = print euler

File: Code/Python/problem005/euler005.py
#!/usr/bin/env python
from __future__ import print_function
import numpy as np

testList = np.array([7, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20])


def euler(testnum):
    while any(np.mod(testnum, testList)):
        testnum += 1
    return testnum


if __name__ == '__main__':
    answer = euler(20)
    print(answer)
